lib/forest_liana/bootstraper.rb

```diff
@@ -32,8 +32,11 @@ def perform
     private
 
     def fetch_sti_models(model)
-      type_field = model.columns.find { |c| c.name == 'type' }
-      if type_field
+      inheritance_field = model.columns.find {
+        |c| c.name == model.inheritance_column || c.name == 'type'
+      }
```

Comment:
It needs a dedicated change in the changelog to explain that the liana support overriden STI type fields